docs/reference/filters.md

```diff
@@ -286,6 +286,28 @@ corsOrigin("https://www.example.org")
 corsOrigin("https://www.example.org", "http://localhost:9001")
 ```
 
+!!! note
+    To enable CORS functionality, you must create a dedicated route that handles `OPTIONS` requests.
+
+Example:
+```
+main_route:
+PathSubtree("/")
+-> corsOrigin()
+-> setResponseHeader("Access-Control-Allow-Credentials", "true")
+-> setResponseHeader("Access-Control-Allow-Methods", "GET,HEAD,OPTIONS,POST,PUT,PATCH,DELETE")
```

Comment:
Lets add spaces after comma between the values to match [the docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Access-Control-Allow-Methods)

Reply:
I think we also indent `->` with spaces elsewhere for readability.